src/applications/personalization/rated-disabilities/rated-disabilities-entry.jsx

```diff
@@ -1,4 +1,5 @@
 import './sass/rated-disabilities.scss';
+import './sass/circular-progress.scss';
```

Comment:
Since we're not using the circular progress bar anymore, this can probably be removed.

Reply:
derp. Removed with commit 0e211ef